geode-cq/src/distributedTest/java/org/apache/geode/cache/query/dunit/QueryMonitorDUnitTest.java

```diff
@@ -383,6 +430,20 @@ private static void executeQuery() {
         verifyException(e);
       }
     }
+
+    final String queryString = "SELECT DISTINCT * FROM /exampleRegion p WHERE p.id = $1";
+    final Query query = queryService.newQuery(queryString);
+
+    try {
+      for (int i = 0; i < 100; ++i) {
+        final Object[] params = new Object[1];
+        params[0] = i;
+
+        query.execute(params);
+      }
+    } catch (final Exception e) {
+      verifyException(e);
+    }
```

Comment:
Why was this extra query logic added?

Reply:
We didn't have any coverage for bind queries in the QueryMonitorDUnitTest, which is where the customer issue was.  Adding this logic technically didn't expose any new issues, but it does now exercise a code path that wasn't covered before.